src/android/run.ts

```diff
@@ -50,7 +58,9 @@ export async function run(args: string[]) {
 
   onBeforeExit(async () => {
     if (ports) {
-      await unforwardPorts(sdk, device, ports);
+    ports.map(async (port: Ports) => {
```

Comment:
You should `await Promise.all(ports.map( ... ))` or the process may exit before the ports are freed